src/components/ItemModal.ts

```diff
@@ -112,38 +112,77 @@ export class ItemModal extends Modal {
         );
     }
 
-    new Setting(contentEl)
-    .setName("Do Date")
-    .addMomentFormat((cb) =>
-        cb
-        .setDefaultFormat(this.dateFormat)
-        .setValue(this.successPlanItem.do_date != "" ? this.successPlanItem.do_date.format(this.dateFormat) : "")
-        .onChange(async (val) => {
-            this.successPlanItem.do_date = moment(val);
-        })
-    );
+    // Do Date Setting 
```

Comment:
Please move this to a new pull request that's specifically for the "Improve how dates are selected via the modal" ticket